keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/TimeWindowConstraint.kt

```diff
@@ -31,6 +31,15 @@ data class TimeWindowConstraint(
   }
 }
 
+/**
+ * Numeric day and hour representation of a time window.
+ * This format is consistent regardless of how the window is defined in a string.
+ */
+data class TimeWindowNumeric(
+  val days: Set<Int>,
+  val hours: Set<Int>,
+)
```

Comment:
Does this representation allow me to have different hours on different days? Is that even a thing we want to support?

Reply:
Yep! It does. You just have several different windows.

However, this is _not_ user config. This is just what we translate the config into for the ui